### PR TITLE
Tm2 upgrade data service

### PR DESCRIPTION
**Pull request by umarkumark**
- Upgraded DataServices project to .NET-6.0

### CODE REVIEW

File: src/DataServices/DataServices.Models/DataServices.Models.csproj [hunk 1]
```diff
@@ -1,21 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
```
Comment (UKHOGreg) [inline src/DataServices/DataServices.Models/DataServices.Models.csproj:6]: Noticed that we have lost the `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>`
Is there a reason, or did we miss it?

File: src/DataServices/DataServices.UnitTests/DataServices.UnitTests.csproj [hunk 2]
```diff
@@ -1,31 +1,23 @@
...
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (UKHOGreg) [inline src/DataServices/DataServices.UnitTests/DataServices.UnitTests.csproj:9]: Noticed that we have lost the `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>`
Is there a reason, or did we miss it?

File: src/DataServices/DataServices/DataServices.csproj [hunk 3]
```diff
@@ -1,52 +1,45 @@
...
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <NoWarn>1701;1702;0472</NoWarn>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (UKHOGreg) [inline src/DataServices/DataServices/DataServices.csproj:13]: Noticed that we have lost the `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>`
Is there a reason, or did we miss it?

File: src/Databases/WorkflowDatabase.Tests/WorkflowDatabase.Tests.csproj [hunk 4]
```diff
@@ -1,22 +1,23 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```
Comment (UKHOGreg) [inline src/Databases/WorkflowDatabase.Tests/WorkflowDatabase.Tests.csproj:6]: I thought we were going to have `nullable` as disabled?

File: src/Databases/WorkflowDatabase.EF/WorkflowDatabase.EF.csproj [hunk 5]
```diff
@@ -1,29 +1,16 @@
...
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (UKHOGreg) [inline src/Databases/WorkflowDatabase.EF/WorkflowDatabase.EF.csproj:8]: Noticed that we have lost the `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>`
Is there a reason, or did we miss it?

**Review by UKHOGreg — CHANGES_REQUESTED** [review]

**Review by uhakdt — APPROVED** [review]

File: src/Databases/WorkflowDatabase.Tests/WorkflowDatabase.Tests.csproj (hunk 4 above)
Comment (SamirHassoun) [reply src/Databases/WorkflowDatabase.Tests/WorkflowDatabase.Tests.csproj]: Fixed, thanks

File: src/DataServices/DataServices.Models/DataServices.Models.csproj (hunk 1 above)
Comment (SamirHassoun) [reply src/DataServices/DataServices.Models/DataServices.Models.csproj:6]: We agreed not add TreatWarningsAsErrors for now

Comment (UKHOGreg) [reply src/DataServices/DataServices.Models/DataServices.Models.csproj:6]: We will revisit this in another PBI

File: src/DataServices/DataServices/DataServices.csproj (hunk 3 above)
Comment (SamirHassoun) [reply src/DataServices/DataServices/DataServices.csproj:13]: We agreed not add TreatWarningsAsErrors for now

File: src/DataServices/DataServices.UnitTests/DataServices.UnitTests.csproj (hunk 2 above)
Comment (UKHOGreg) [reply src/DataServices/DataServices.UnitTests/DataServices.UnitTests.csproj:9]: We will revisit this in another PBI

File: src/Databases/WorkflowDatabase.EF/WorkflowDatabase.EF.csproj (hunk 5 above)
Comment (SamirHassoun) [reply src/Databases/WorkflowDatabase.EF/WorkflowDatabase.EF.csproj:8]: We agreed not add TreatWarningsAsErrors for now

File: src/DataServices/DataServices/DataServices.csproj (hunk 3 above)
Comment (UKHOGreg) [reply src/DataServices/DataServices/DataServices.csproj:13]: We will revisit this in another PBI

File: src/DataServices/DataServices.UnitTests/DataServices.UnitTests.csproj (hunk 2 above)
Comment (SamirHassoun) [reply src/DataServices/DataServices.UnitTests/DataServices.UnitTests.csproj:9]: We agreed not add TreatWarningsAsErrors for now

**Review by UKHOGreg — APPROVED** [review]

**Pull request merged**